src/v/bytes/tests/iobuf_tests.cc

```diff
@@ -491,4 +491,14 @@ SEASTAR_THREAD_TEST_CASE(trim_front) {
         buf.trim_front(101);
```

Comment:
I think that the explanation in this commit message should be the commit message for the commit that fixes the bug. Since this patch comes first, it would trigger the bug. I'd recommend either (1) combining the commits or (2) ordering this commit after the other to preserve testing bisectability.

Reply:
got it